Sources/Sentry/SentryHub.m

```diff
@@ -28,6 +28,7 @@ - (instancetype)initWithClient:(SentryClient *_Nullable)client
         [self bindClient:client];
         self.scope = scope;
         _sessionLock = [[NSObject alloc] init];
+        _installedIntegrations = [[NSMutableArray alloc] init];
```

Comment:
L: Is this a fix found out by tests or how this is part of this PR?

Reply:
Yes, I found this issue when writing tests for SentrySDK. I thought I just add it here or would you prefer another PR for it?

Reply:
its fine to be here IMO, just would like to understand such change, nice catch